utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -673,6 +676,57 @@ where
     }
 }
 
+#[derive(PartialEq, Eq, Debug, Clone, Copy)]
+pub struct CodePointMapRange {
+    start: u32,
+    end: u32,
+    value: u32,
+}
+
+impl CodePointMapRange {
```

Comment:
Suggestion: make it a naked struct with public fields and name it something more general (as suggested above).  We don't need these extra functions.

Reply:
naked struct is ok with me

Reply:
Done.